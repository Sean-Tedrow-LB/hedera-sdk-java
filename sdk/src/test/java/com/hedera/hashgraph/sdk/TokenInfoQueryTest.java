package com.hedera.hashgraph.sdk;

import io.github.jsonSnapshot.SnapshotMatcher;
import org.junit.AfterClass;
import org.junit.jupiter.api.BeforeAll;
import org.junit.jupiter.api.Test;

public class TokenInfoQueryTest {
    @BeforeAll
    public static void beforeAll() {
        SnapshotMatcher.start();
    }

    @AfterClass
    public static void afterAll() {
        SnapshotMatcher.validateSnapshots();
    }

    @Test
    void shouldSerialize() {
        SnapshotMatcher.expect(new TokenInfoQuery()
            .setTokenId(TokenId.fromString("0.0.5005"))
            .setMaxQueryPayment(Hbar.fromTinybars(100_000))
            .toString()
        ).toMatchSnapshot();
    }
}
