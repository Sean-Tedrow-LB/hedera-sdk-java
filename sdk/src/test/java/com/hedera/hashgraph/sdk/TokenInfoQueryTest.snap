com.hedera.hashgraph.sdk.TokenInfoQueryTest.shouldSerialize=[
  "\ntoken_get_info {\n  header {\n  }\n  token {\n    realm_num: 0\n    shard_num: 0\n    token_num: 5005\n  }\n}"
]